### ISSUE TITLE
Add @microsoft/applicationinsights-web to peer dependencies

### ISSUE THREAD

**Issue by jlilly** [microsoft/applicationinsights-angularplugin-js#107]
`ApplicationInsights`  from `@microsoft/applicationinsights-web` is required to configure this plugin, yet the dependency is not stated as a peer dependency. This makes upgrading a pain.

**Comment by MSNev** [comment]
This extension does not have a dependency on the `ApplicationInsights` from `@microsoft/applicationinsights-web`

This extension can be used without the main Sku (from  `@microsoft/applicationinsights-web`), its dependencies are the same as those used by the main Sku.

While the easiest way to consume and use this extension is via adding it to an `ApplicationInsights` from `@microsoft/applicationinsights-web` instance that is not required.

> This makes upgrading a pain.

What exactly do you mean by this? We have not yet released a version of this extension which explicitly uses the new v3.x version of ApplicationInsights (this is planned to occur this month (June, 2023)

**Comment by stewart-southwell** [comment]
A related issue seems to be that the main SKU has a dependency on **version 3.0.2** of the  `@microsoft/applicationinsights-analytics-js` package.  However,  the `@microsoft/applicationinsights-angularplugin-js` package has a peer dependency on **version 2.8.14** of the `@microsoft/applicationinsights-analytics-js` package.  This poses a problem when you have the main SKU and the plugin as a dependency in your project as it is not able to resolve the dependency.  If you run `npm install` you will receive the following error.   

![image](https://github.com/microsoft/applicationinsights-angularplugin-js/assets/42212376/4f006247-f9c6-4322-8fdb-eb685417a737)

**Comment by MSNev** [comment]
Yes, we have not yet released a version of the angular plugin that supports v3.x (this is scheduled in the coming weeks) and the plan is to release by the end of the month (or early next month)
